### ISSUE TITLE
Complain of missing translation

### ISSUE THREAD

**Issue by kjeremy** [codegram/date_validator#44]
I get the following:

Translation missing: en.activerecord.errors.models.affiliation.attributes.start.before

When I have this in my model:

class Affiliation
  validates :start, :date => { :before => :end }
end

**Comment by Uepsilon** [comment]
just stumbled across this. Since @gauthier-delacroix solution does not work with ruby > 2.1 and ruby 4, here is the adapted solution for ruby 2.1.5 with rails 4.x

Single: 
```ruby
I18n.load_path += "#{Gem.loaded_specs['date_validator'].full_gem_path}/locales/en.yml"
```

All
```ruby
I18n.load_path += Dir[File.expand_path( '*.yml', "#{Gem.loaded_specs['date_validator'].full_gem_path}/locales").to_s]
```

**Comment by vjustov** [comment]
This is fixed in #60

**Comment by oriolgual** [comment]
Released https://github.com/codegram/date_validator/tree/v0.8.0 with this fix.
